### DISCUSSION TITLE
Add docs on how to start rsync script automatically

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#1105] [Ideas] [answered]
### How to use GitHub
* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Is your feature request related to a problem? Please describe. 

Whenever the hard disk is connected for backups, the script has to be run manually, it would be easier if this could be automated.  Would be a cool feature and would definitely have a couple of users.

### Describe the solution you'd like

Whenever the external hard drive is plugged into the server, the backup with rsync should start automatically and when it is finished, a message is sent via Nextcloud that the backup is complete.

### Describe alternatives you've considered

You could leave it as is but it would be nice if this could be added as an option in the rsync script.

### Additional context

A very good solution is described here:

"
BUS=="usb", SYSFS{serial}=="XXXXXXXXXXX", KERNEL=="sd*", NAME="%k", SYMLINK+="backup%n"
ACTION=="add", SYSFS{serial}=="XXXXXXXXXXX", KERNEL=="sd*", RUN="/bin/backup.sh"
ACTION=="remove", KERNEL=="sd*", RUN+="/bin/backup_remove.sh"
"

https://www.linuxforen.de/forums/showthread.php?266406-Wenn-USB-Stick-erkannt-automatisch-ein-Sktipt-starten

**Comment by szaimen** [comment, marked as answer]
Some help on this like a PR is very appreciated!
